Intro.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE HTML>
+<head>
+	<title>About</title>
+</head>
+<body>
+</html>
```

Comment:
You should not close </html> till your page is finished, <body> also part of <html>